Komodo/Assets/Packages/KomodoCore/Hidden~/WebGLTemplates/KomodoWebXRFullView2020/relay.js

```diff
@@ -1,9 +1,11 @@
 // 'esversion: 6'
 
-// Replace these with your own server's URLs.
+// Remember: also change Assets\Packages\KomodoCore\Hidden~\WebGLTemplates\KomodoWebXRFullView2020\relay.js
```

Comment:
@parseccentric Probably need to change this so module developers don't think they need to edit this if they're just changing the configuration variables (RELAY_BASE_URL, etc.) and not changing the functionality below those config variables.